README.md

```diff
@@ -39,9 +44,16 @@ flags menu:
 - transform/warp                // Toggle flag for transforming/warping
 - wolf sense                    // Toggle flag for having wolf sense
 
+dungeon flags menu:
+- have map                      // Toggle dungeon map for current dungeon
+- have compass                  // Toggle compass for current dungeon
+- have boss key                 // Toggle boss key for current dungeon
+- have small keys               // Toggle small keys for current dungeon
+
 inventory menu:
 - item wheel                    // Can set the 24 item wheel slots to any item
-- pause menu (TBD)              // Can modify the pause menu collection (not implemented yet)
+- pause menu                    // Can modify the pause menu collection
```

Comment:
Similarly, these would probably be better as "Set the 24 item wheel slots" and "Modify the pause menu collection". I realize that you didn't alter the above line. The one below this could be "Modify ammo/collectible amounts".